lib/base-cmd.js

```diff
@@ -129,6 +131,63 @@ class BaseCommand {
     }
   }
 
+  // Checks the devEngines entry in the package.json at this.localPrefix
+  async checkDevEngines () {
+    const force = this.npm.flatOptions.force
+
+    const { devEngines } = await require('@npmcli/package-json')
+      .normalize(this.npm.config.localPrefix)
+      .then(p => p.content)
+      .catch(() => ({}))
```

Comment:
Can we easily log the error? Might be useful for bug reports about `devEngines` not being checked.

Reply:
There are no errors that will happen here specific to checking devEngines.  Either there's a valid package.json here which will also be parsed later or there isn't.  If the latter parsing fails that's where logging happens.